x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/Security.java

```diff
@@ -274,7 +274,7 @@
                     s -> s.keySet().contains(SecurityField.setting("audit.outputs"))
                             ? Collections.emptyList()
                             : Collections.singletonList(LoggingAuditTrail.NAME),
-                    Property.NodeScope);
+                    Property.NodeScope, Property.Deprecated);
```

Comment:
As above, is this really what we want?
I'm not totally against it, but the description of the PR, and the docs changes don't indicate that we're deprecating the `outputs` setting. Since there's only 1 non-deprecated value, and that's the default it makes some sense to just say the setting is obsolete, but the rest of this PR doesn't say that.

Reply:
Okay, I will update the description. Sorry if it came out as a "hidden" change!